tests/test_models.py

```diff
@@ -136,6 +136,20 @@ def test_deserialize_wrong_type_name(self):
         record = Inventory()
         self.assertRaises(DataValidationError, record.deserialize, data)
 
+    def test_deserialize_wrong_type_active(self):
+        """It should not deserialize inventory with wrong type of active"""
+        data = {
+            "active": "wrong_type",
+            "product_id": 1,
+            "name": "laptop",
```

Comment:
Get the latest master which does not have reorder_quantity and restock_level

Reply:
it will auto merge and will remove conflicts after review